pkg/mimirtool/commands/alerts.go

```diff
@@ -104,27 +113,39 @@ func (a *AlertmanagerCommand) getConfig(k *kingpin.ParseContext) error {
 	return p.PrintAlertmanagerConfig(cfg, templates)
 }
 
-func (a *AlertmanagerCommand) loadConfig(k *kingpin.ParseContext) error {
+func (a *AlertmanagerCommand) prepareAlertManagerConfig(k *kingpin.ParseContext) (error, string, map[string]string) {
```

Comment:
[nit] WDYT about calling it "read"? We're basically reading it from file.

```suggestion
func (a *AlertmanagerCommand) readAlertmanagerConfigFromFile(k *kingpin.ParseContext) (error, string, map[string]string) {
```

Reply:
Agreed. And done.